samples/drivers/video/capture/boards/arduino_nicla_vision_stm32h747xx_m7.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 Felipe Neves
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+		zephyr,shell-uart = &cdc_acm_uart0;
+		zephyr,cdc-acm-uart0 = &cdc_acm_uart0;
```

Comment:
Do you know where `zephyr,cdc-acm-uart0` chosen node is used? I cannot find it anywhere.
If nowhere, the could be removed.
I also see it here: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arduino/portenta_h7/arduino_portenta_h7_stm32h747xx_m7.dts#L20 but maybe it is a typo?

Reply:
i get the reference from the Arduino Opta, which defaults its console to USB.

Reply:
In my understanding, this has no effect and can be left out, but does not harm either. Maybe some external code relies on this for the other boards it came from.